workbench-cypress/cypress/e2e/database/spreadsheet.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { newClickFlow, newExpectationWithClickFlows } from "@utils/helpers";
+import { runTests } from "@utils/index";
+import {
+  beVisible,
+  beVisibleAndContain,
+  checkValueInGridTests,
+  getTypeInGridTests,
+  shouldBeVisible,
+  shouldFindAndContain,
+} from "@utils/sharedTests/sharedFunctionsAndVariables";
+
+const pageName = "Edit table with spreadsheet editor";
+const dbName = "us-jails";
+const currentPage = `/database/${dbName}`;
+const tableToEdit = "incidents";
+
+const newRows = [
+  ["100", "2020-03-09", "2020-03-10", "10", "300", "0"],
+  ["101", "2020-03-10", "2020-03-11", "0", "3", "0"],
+];
+
+describe(`${pageName} renders expected components on different devices`, () => {
```

Comment:
I think we have this full statement in the util that runs the tests?
```suggestion
describe(pageName, () => {
```